onnxruntime/python/tools/quantization/shape_inference.py

```diff
@@ -75,20 +75,24 @@ def quant_pre_process(
         temp_path = Path(quant_tmp_dir)
         model = None
 
-        if not skip_symbolic_shape:
+        if enable_symbolic_shape:
             logger.info("Performing symbolic shape inference...")
-            loaded_model = input_model if isinstance(input_model, onnx.ModelProto) else onnx.load(input_model)
-            model = SymbolicShapeInference.infer_shapes(
-                loaded_model,
-                int_max,
-                auto_merge,
-                guess_output_rank,
-                verbose,
-            )
+            try:
+                loaded_model = input_model if isinstance(input_model, onnx.ModelProto) else onnx.load(input_model)
+                model = SymbolicShapeInference.infer_shapes(
+                    loaded_model,
+                    int_max,
+                    auto_merge,
+                    guess_output_rank,
+                    verbose,
+                )
+            except Exception:
+                logger.error("ONNX Runtime symbolic shape inference failed! Consider rerun without it.")
+                logger.error(traceback.format_exc())
```

Comment:
should the exception be rethrown or is it intentionally only logged before continuing?

Reply:
it is intentionally to logged instead of exit.